tabular/tests/unittests/test_tabular.py

```diff
@@ -111,6 +111,7 @@ def test_advanced_functionality():
     savedir_predictor_original = savedir + 'predictor/'
     predictor: TabularPredictor = TabularPredictor(label=label, path=savedir_predictor_original).fit(train_data)
     leaderboard = predictor.leaderboard(data=test_data)
+    predictor.plot_ensemble_model()
```

Comment:
we need to install pygraphviz for unit tests to work, it isn't installed by default.

Reply:
I see. it might be tricky to install this in our CI env as it requires some C++ library. I'll take a look